.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+name: "Close stale issues"
+
+on:
+  schedule:
+  - cron: "0 * * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: >
+            This issue has been automatically marked as stale because it has
+            not had recent activity. It will be closed if no further activity
+            occurs.
```

Comment:
This message was loosely adapted from the stale action in an [Ansible module](https://github.com/elastic/ansible-elasticsearch/blob/master/.github/stale.yml). I'm going for clarity and general applicability in our repos.